### PR TITLE
Clean up Event2Mind output.

### PR DESCRIPTION
**Pull request by brendan-ai2**
- Remove `@@UNKNOWN@@` token.
- Replace `none` with a nicer descriptor.

### CODE REVIEW

**Comment by brendan-ai2** [comment]
FYI, @maartensap

File: demo/src/components/Event2MindComponent.js [hunk 1]
```diff
@@ -127,6 +130,19 @@ class Event2MindInput extends React.Component {
...
 
+// Remove the special "@@UNKNOWN@@" token from model output, replace "none", join tokens.
+function processTokens(tokens, noneReplacement) {
+  return tokens.filter(target => {
+    return target.length !== 1 || target[0] !== "@@UNKNOWN@@";
+  }).map(target => {
+    if (target.length === 1 && target[0] === "none") {
+      return <em> {noneReplacement} </em>;
+    } else {
+      return target.join(" ");
```
Comment (schmmd) [inline demo/src/components/Event2MindComponent.js]: We didn't have this join before--are you sure it's OK now?

File: demo/src/components/Event2MindComponent.js [hunk 2]
```diff
@@ -127,6 +130,19 @@ class Event2MindInput extends React.Component {
   }
 }
 
+// Remove the special "@@UNKNOWN@@" token from model output, replace "none", join tokens.
+function processTokens(tokens, noneReplacement) {
+  return tokens.filter(target => {
+    return target.length !== 1 || target[0] !== "@@UNKNOWN@@";
```
Comment (schmmd) [inline demo/src/components/Event2MindComponent.js]: DeMorgan's rule would help me out a bit here ;-)

But that might just be me.

!(target.length === 1 && target[0] === "@@UNKNOWN@@")

File: demo/src/components/Event2MindComponent.js (hunk 1 above)
Comment (schmmd) [reply demo/src/components/Event2MindComponent.js]: nevermind...

Comment (brendan-ai2) [reply demo/src/components/Event2MindComponent.js]: 👍

File: demo/src/components/Event2MindComponent.js (hunk 2 above)
Comment (brendan-ai2) [reply demo/src/components/Event2MindComponent.js]: Done!

**Comment by brendan-ai2** [comment]
Comments addressed. Thanks!

**Pull request merged**